lib/shortenator.rb

```diff
@@ -28,16 +28,23 @@ def configure
     def search_and_shorten_links(
       text,
       domains = config.domains,
-      ignore_200_check: config.ignore_200_check
+      ignore_200_check: config.ignore_200_check,
+      tags: config.tags,
+      additional_tags: []
     )
+      @all_tags = tags + additional_tags
```

Comment:
Instance variables in class methods are going to likely be a source of bugs down the line, as this will persist between invocations of `search_and_shorten_links`. It looks like you can probably make this a local variable if you update the `shorten_link` private method to accept a `tags` keyword argument for the tags to use.